ecr/resolver.go

```diff
@@ -36,7 +36,10 @@ import (
 	"github.com/pkg/errors"
 )
 
-var unimplemented = errors.New("unimplemented")
+var (
+	ErrInvalidManifest = errors.New("invalid manifestr")
```

Comment:
Looks like an extra key press snuck in here.

```suggestion
	ErrInvalidManifest = errors.New("invalid manifest")
```

Reply:
Oops. Fixed in the last revision.